index.js

```diff
@@ -56,17 +47,17 @@ app.post('/admin_poll', (req, res) => {
 
   adminPolls[id] = req.body.adminPoll; adminVotes[id] = adminTally
   adminPolls[`${id}`]['refId'] = id; liveAdminPolls[liveId] = adminPolls[id]
```

Comment:
Why do we need a template string for `id`? Couldn't we just use the value?

I just did a quick test in the console and JavaScript seems to coerce the index anyway, so it probably doesn't matter if you use a string or a number.

```js
adminPolls[id]['refId'] = id;
```

Also, the thing where you have two statements on the same line with a semi-colon is confusing to someone who didn't write the code.